src/desktop/apps/personalize/components/RoutesWithProgressBar.js

```diff
@@ -15,27 +16,28 @@ const STEPS = [
 ]
 
 export class RoutesWithProgressBar extends React.Component {
```

Comment:
Sorry for all the confusion, but the `<RoutesWithProgressBar>` component was removed as part of https://github.com/artsy/force/pull/2202 since the `<Wizard>` in reaction has `@track` decorators and some tracking code while this one does not. Would you mind rebasing?